example/index.html

```diff
@@ -51,6 +51,23 @@
                    template="/src/invenio-search-js/templates/searchBar.html"
                    placeholder="Start typing">
                   </invenio-search-bar>
+                  <div class="text-right">
+                    <!-- SORT ORDER -->
+                    <invenio-search-sort-order
+                     label="Z->A"
+                     sort-key="sort"
+                     template="/src/invenio-search-js/templates/extras/toggleButton.html"
```

Comment:
Not sure if I'm doing something wrong, but these templates seems to not be copied into ``dist`` folder when you do a build.